lookout/style/format/benchmarks/release_v0.1/30-seconds-of-code

```diff
@@ -0,0 +1,82 @@
+[1;36mINFO[0m:94f5:QualityReportAnalyzer:[0mClassification report:
+                   precision    recall  f1-score   support
```

Comment:
Should be formatted in Markdown

Reply:
yes, it will be formatted in markdown. Just I need to calculate all these reports one more time.